### PR TITLE
'Solution'

### PR DESCRIPTION
**Pull request by sind14**

### CODE REVIEW

File: app/cinema/hall.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+
+class CinemaHall:
+    def __init__(self, number: int) -> None:
+        self.number = number
+
+    def movie_session(
+            self,
+            movie_name: str,
+            customers: list,
```
Comment (EdAlekseiev) [inline app/cinema/hall.py:12]: List of what? Specify the type of list

File: app/main.py [hunk 2]
```diff
@@ -1,6 +1,17 @@
...
 
+Customer("Anton", "apple")
+Cleaner("Anton")
 
-def cinema_visit(customers: list, hall_number: int, cleaner: str, movie: str):
-    # write you code here
-    pass
+
+def cinema_visit(
+        customers: list,
```
Comment (EdAlekseiev) [inline app/main.py:11]: List of what? Specify the type of list

File: app/main.py [hunk 3]
```diff
@@ -1,6 +1,17 @@
...
+from app.people.cinema_staff import Cleaner
 
+Customer("Anton", "apple")
+Cleaner("Anton")
 
-def cinema_visit(customers: list, hall_number: int, cleaner: str, movie: str):
-    # write you code here
-    pass
+
+def cinema_visit(
```
Comment (EdAlekseiev) [inline app/main.py:10]: I think, that you must convert `customers` from `dict` to `Customer` inside this function instead of `movie_session`

**Review by EdAlekseiev — CHANGES_REQUESTED** [review]

**Review by Bohdan-Salamakha — CHANGES_REQUESTED** [review]
don't work on the master branch - it's a bad practice
https://github.com/mate-academy/py-task-guideline/blob/main/README.md#:~:text=git%20checkout%20%2Db%20develop

**Pull request closed without merging**